### PR TITLE
fix: #1509 Replace OIDC SSO playground url

### PR DESCRIPTION
**Pull request by craigyu**
Closes: #1509 

- Replaced old OIDC debugger url with playground url

### CODE REVIEW

File: infrastructure/server/variables_provided.tf [hunk 1]
```diff
@@ -372,4 +372,10 @@ variable "dev_pr_url_count" {
...
   default = 50
-}
\ No newline at end of file
+}
+
+variable "oidc_sso_playground_url" {
+  description = "OIDC SSO Playground for debugging congnito login"
+  type = string
+  default = "https://sso-playground.apps.gold.devops.gov.bc.ca"
+}
```
Comment (ianliuwk1019) [inline infrastructure/server/variables_provided.tf:381]: It's not clear in this file how it is organized. Would you please place this variable under line 215 or line 55 section? Thanks!

File: infrastructure/server/oidc_clients_fam.tf [hunk 2]
```diff
@@ -3,13 +3,17 @@ resource "aws_cognito_user_pool_client" "fam_console_oidc_client" {
...
     [
+      var.oidc_sso_playground_url,
       "${aws_api_gateway_deployment.fam_api_gateway_deployment.invoke_url}/docs/oauth2-redirect",
       "${aws_api_gateway_stage.admin_management_api_gateway_stage.invoke_url}/docs/oauth2-redirect"
     ]
   ))
-  logout_urls                                   = var.fam_logout_urls
+  logout_urls                                   = concat(
+    var.fam_logout_urls,
+    [var.oidc_sso_playground_url])
```
Comment (ianliuwk1019) [inline infrastructure/server/oidc_clients_fam.tf:16]: Hi @MCatherine1994 , sorry, I am wondering, does logout_urls need to include `oidc_sso_playground_url`? Does `oidc_sso_playground_url` provide logout?

Comment (MCatherine1994) [reply infrastructure/server/oidc_clients_fam.tf:16]: yes they provide logout, but we might don't need to test the logout. It doesn't hurt to include in the logout urls.

File: infrastructure/server/variables_provided.tf (hunk 1 above)
Comment (craigyu) [reply infrastructure/server/variables_provided.tf]: https://github.com/bcgov/nr-forests-access-management/pull/1565/commits/e226e2c58c4f46adcb2424c4fa51bdb6e2611f16 Done

File: infrastructure/server/oidc_clients_fam.tf (hunk 2 above)
Comment (ianliuwk1019) [reply infrastructure/server/oidc_clients_fam.tf:16]: Cognito logout seems confusing, I forgot if Cognito enforce them or only on callback. Sure, then leave it for now. Thanks Catherine.

**Review by ianliuwk1019 — APPROVED** [review]
Thanks Craig!

**Pull request merged**